index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Newton Tribute Page</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <div class="container">
+      <div id="main">
+        <header>
+            <h1 id="title" class="text-center">Newton Tribute Page</h1>
+        </header>
+        <figure id="img-div">
+          <figcaption id="img-caption" class="text-center">Isacc Newton</figcaption>
+          <img id="image" src="images/issacn.jpg" alt="Newton">
+        </figure>
+        <div id="tribute-info">
+            <p style="text-align: justify;">
```

Comment:
Please move all styles to a styles.css file to follow the coding best practices. It improves readability with cleaner code.👀